src/app/components/Headline/index.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import styled from 'styled-components';
 import { extractText } from '../../helpers/blockHandlers';
-import { textPropTypes, textDefaultPropTypes } from '../../models/proptypes';
+import { textDefaultPropTypes } from '../../models/proptypes';
```

Comment:
Is there a reason why `textDefaultPropTypes` has not also been moved into a file in the structure `../../models/propTypes/text`? Or were you thinking of doing this in a separate PR, to keep PRs granular?

Reply:
Simply because I hadn't properly considered default prop types :)

Reply:
:) OK. It would be good to put the default prop types in the same structure, since otherwise it can be quite confusing to know where to make changes.

Reply:
Maybe we should open an issue to cover the work @sareh is describing so it's not lost in this PR thread

Reply:
I've opened an issue here: https://github.com/bbc/simorgh/issues/279